platform/contracts/admin/src/endpoints.rs

```diff
@@ -49,13 +48,12 @@ pub fn instantiate(
     state_contracts::store(deps.storage, contracts).map(|()| response::empty_response())
 }
 
-#[cfg(feature = "migrate")]
 #[entry_point]
 pub fn migrate(
     deps: DepsMut<'_>,
     env: Env,
     MigrateMsg {
-        reserve_contracts,
+        dex_data,
```

Comment:
do you believe that all existing protocols share the same dex_data? Or I am mistaken here?

Reply:
disregard it, just saw the type